Timed out waiting for /ft_sensor/raw topic

Running the hybrid force control example [here](https://crigroup.gitbooks.io/osrobotics/content/control/examples.html) will fail with the following error:

```
nicholas@zbook:~/osr_ws/src/osr_course_pkgs/osr_examples/scripts$ ./force_example.py                 
[INFO] [1565329048.020525, 33.486000]: JointPositionController initialized. ns: 
[INFO] [1565329048.266422, 33.730000]: GripperCommandAction initialized. ns: /
[ERROR] [1565329051.275560, 36.736000]: Timed out waiting for /ft_sensor/raw topic
[INFO] [1565329056.633876, 42.087000]: Going down
Traceback (most recent call last):
  File "./force_example.py", line 114, in <module>
    We = ft_sensor.get_filtered_wrench() - wrench_offset
TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType'
```

--- a/dependencies.rosinstall
+++ b/dependencies.rosinstall
@@ -2,3 +2,4 @@
 - git: {local-name: gazebo_ros_link_attacher, uri: 'https://github.com/pal-robotics/gazebo_ros_link_attacher.git',  version: master}
 - git: {local-name: openrave_catkin,          uri: 'https://github.com/crigroup/openrave_catkin.git',               version: master}
 - git: {local-name: robotiq,                  uri: 'https://github.com/crigroup/robotiq.git',                       version: master}
+- git: {local-name: gazebo_ros_pkgs,          uri: 'https://github.com/ros-simulation/gazebo_ros_pkgs.git',         version: kinetic-devel}

